app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -290,4 +290,8 @@
   <dimen name="coming_soon_topic_promoted_list_view_padding_start">0dp</dimen>
   <dimen name="coming_soon_topic_Promoted_list_view_padding_end">0dp</dimen>
 
+  <!--  Topic practice footer view-->
```

Comment:
Nit spacing missing
`<!--  Topic practice footer view -->`

Reply:
@rt4914  I have added spaces and capitalized this.